cli/cmd/tapRunner.go

```diff
@@ -261,6 +261,7 @@ func portForwardApiPod(ctx context.Context, kubernetesProvider *kubernetes.Provi
 					}
 					rlog.Debugf("Sending get request to %v\n", u.String())
 					if response, err := http.Get(u.String()); err != nil || response.StatusCode != 200 {
+						fmt.Printf(u.String())
```

Comment:
Debugging leftover?

If this is a print you want to keep, then it needs to be clear what is being printed here.
What is u?
```go
fmt.Printf("Sending traffic to: %s\n", u.String())
```